### DISCUSSION TITLE
What is hoppscotch.io

### DISCUSSION THREAD

**Discussion by ghost** [github/india#24] [Open Source] [answered]
what is hoppscotch.io

**Comment by manavmodi22** [comment]
It started out as an idea of an API testing tool. Now they are offering a lot more stuff. Check them out here : [https://github.com/hoppscotch](https://github.com/hoppscotch)

**Comment by fabianferno** [comment]
Hoppscotch.io is an open-source API development tool which helps you build and test APIs. It's ability to make various types of HTTP requests like GET, POST, PUT, PATCH, etc and create documentations for your collection of APIs, query graphQL and many other awesome features comes as a one-stop solution for writing APIs for your projects

**Comment by liyasthomas** [comment, marked as answer]
Hi I'm Liyas - co-creator of Hoppscotch project.

Hoppscotch is an open-source API development ecosystem.

Hoppscotch support testing and debugging:
- RESTful API endpoints
- GraphQL API endpoints
- WebSockets
- SSE endpoints
- Socket.IO endpoints
- MQTT brokers
Plus,
- Generate self-hostable API documentation
- Save and share APIs in collections and folders
- Environment variables
- Pre-request script
- Post-request tests
- Team collaboration (beta)
- Workspaces (closed beta)

Web app: https://hoppscotch.io
GitHub: https://github.com/hoppscotch/hoppscotch
